deltachat-ios/Controller/ChatViewController.swift

```diff
@@ -274,7 +274,7 @@ class ChatViewController: MessagesViewController {
 
         let badge: InitialsBadge
         if let image = chat.profileImage {
-            badge =  InitialsBadge(image: image, size: 28)
+            badge =  InitialsBadge(image: image, size: 28, accessibilityLabelText: chat.name)
```

Comment:
this is the avatar in the title bar above an opened chat?

a tap on it opens the profile - so maybe better just use `menu_view_profile` / "View profile"?